terasoluna-gfw-parent/pom.xml

```diff
@@ -515,7 +515,7 @@
     <!-- == Other == -->
     <jakarta-el.version>5.0.1</jakarta-el.version>
     <jakarta-inject.version>2.0.1</jakarta-inject.version>
-    <jakarta-jsp.version>3.1.0</jakarta-jsp.version>
+    <jakarta-jsp.version>3.0.0</jakarta-jsp.version>
```

Comment:
- [ ] jakarta.servlet.jsp-api should be 3.1.0

Reply:
Fixed in the following commit.
https://github.com/terasolunaorg/terasoluna-gfw/pull/1217/commits/bc71e46813aee1b430b24849e1a24cdea8985423